README.md

```diff
@@ -14,6 +14,15 @@ Welcome to the [STACKIT](https://www.stackit.de/en) CLI, a command-line interfac
 This CLI is in a BETA state. More services and functionality will be supported soon.
 Your feedback is appreciated!
 
+<a name="warning-new-stackit-idp"></a>

```

Comment:
The anchor allows opening the README and scroll directly to the warning using https://github.com/stackitcloud/stackit-cli#warning-new-stackit-idp

Reply:
Doesn't markdown already do this? e.g https://github.com/stackitcloud/stackit-cli?tab=readme-ov-file#authentication